### PR TITLE
Add fuse-gulp-rollup-babel

### PR DESCRIPTION
**Pull request by sebbert**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,6 +93,7 @@ You can find more examples in [Fuse official examples list](https://www.fusetool
 - [Fuse Samples](https://github.com/fusetools/fuse-samples) - A collection of smaller examples using the various features of Fuse and FuseJS.
 - [FuseBus](http://tmn.github.io/FuseBus/) - A simple Fuse application for testing and funtimes.
 - [Gulp Procedure](https://github.com/joms/gulp-fuse) - Gulp-procedure for Fuse compiling ES6 into ES5.
+- [Fuse+Gulp+Rollup+Babel](https://github.com/sebbert/fuse-gulp-rollup-babel) - An alternative example of module bundling and ES6->ES5 transpiling.
```
Comment (vinkla) [inline README.md]: 1. Please change `Fuse+Gulp+Rollup+Babel` to `Gulp Rollup Babel`.
2. Please change `ES6->ES5` to `ES6 -> ES5`.

**Comment by vinkla** [comment]
Awesome! I'll merge this as soon as you've looked at my comments.

**Comment by sebbert** [comment]
@vinkla There ya go :)

**Comment by vinkla** [comment]
Thanks!

**Pull request merged**